huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go

```diff
@@ -95,6 +95,15 @@ func ResourceGaussRedisInstanceV3() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"port": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
```

Comment:
when the port is not specified, it will be set to the default value, then the resource will be rebuild as the **ForceNew** field, so please add **Computed** to avoid this situation.